modules/nf-core/mtmalign/align/main.nf

```diff
@@ -0,0 +1,53 @@
+
+
+process MTMALIGN_ALIGN {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/mtm-align:20220104--h4ac6f70_0':
+        'biocontainers/mtm-align:20220104--h4ac6f70_0' }"
+
+    input:
+    tuple val(meta), path('*.pdb', arity: '2..*')
+
+    output:
+    tuple val(meta), path("./mTM_result/*.aln")  , emit: alignment
+    tuple val(meta), path("./mTM_result/${prefix}.pdb")    , emit: structure
+    path "versions.yml"                                 , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    ls *.pdb | sed s/\\ /\\n/ > input_list.txt
+    mtm-align -i input_list.txt -o ${prefix}.pdb
```

Comment:
This should fix the problem with the result.pdb right?

Reply:
yes, this fixes it in a more clean way, I think!